include/lapack/gebd2.hpp

```diff
@@ -0,0 +1,65 @@
+#ifndef GEBD2
```

Comment:
have a look at other <T>LAPACK codes, they all have comments at the start. Please do the same.